lms/templates/static_pdfbook.html

```diff
@@ -48,7 +48,7 @@
   height="1108"
   frameborder="0"
   seamless
-  tabindex="1"
+  tabindex="0"
```

Comment:
Also, the ```<ul>``` on [line 32 of this file](https://github.com/edx/edx-platform/pull/12656/files#diff-b71ac66472a3d3feee823c957a10f4eeR32) should not have a tabindex at all since it is not natively focusable.  The list items themselves have anchor elements that will be what the user will want to focus on.